core/src/test/java/org/apache/druid/java/util/common/DateTimesTest.java

```diff
@@ -37,4 +37,17 @@ public void testCommonDateTimePattern()
       Assert.assertTrue(DateTimes.COMMON_DATE_TIME_PATTERN.matcher(dt.toString()).matches());
     }
   }
+
+  @Test
+  public void testStringTimestampToDateTimeConversion()
+  {
+    Long milis = System.currentTimeMillis();
+    DateTime dt1 = DateTimes.of(milis.toString());
+    Assert.assertTrue(DateTimes.COMMON_DATE_TIME_PATTERN.matcher(dt1.toString()).matches());
```

Comment:
Could you please edit this test to check that the parsed values are actually what we expect (not just that it matches `COMMON_DATE_TIME_PATTERN`). It would probably help to use a constant instead of `System.currentTimeMillis()`.